### DISCUSSION TITLE
IP address in a different subnet

### DISCUSSION THREAD

**Discussion by philippenc** [jaredhendrickson13/pfsense-api#525] [Q&A]
Hello,

Thank you again for your help. My POST requests in the DHCP are working. I just noticed that via the REST API api/v2/services/dhcp_server/static_mapping, it is possible to add an IP address that does not belong to the subnet of the parent_id. This action is not possible from the web interface. Do you plan for the REST API to verify that the IP addresses in the requests match the subnet of the parent_id in the future?

Thank you.

**Comment by jaredhendrickson13** [comment]
Looks like the validation for this was not ported correctly from v1, it checks for conflicts but skips checking that that IP is within the interface's subnet. This can be fixed pretty easily. I've opened issue #526 for this. Feel free to follow that issue for updates.

Thanks!
